ansible/roles/commcarehq/templates/logrotate.conf.j2

```diff
@@ -0,0 +1,29 @@
+# see "man logrotate" for details
+# rotate log files weekly
+weekly
+
+# packages drop log rotation information into this directory
+include /etc/logrotate.d
+
+# no packages own wtmp, or btmp -- we'll rotate them here
+/var/log/wtmp {
+    missingok
+    monthly
+    create 0664 root utmp
+    rotate 1
+}
+
+/var/log/btmp {
+    missingok
+    monthly
+    create 0660 root utmp
+    rotate 1
+}
+
+{{ log_home }}/*.log {
+    weekly
+    missingok
+    compress
+    delaycompress # compress on second rotation
+    olddir {{ rotated_log_home }}
```

Comment:
i think it actually might be best to use the log rotate python class rather than this. a lot of the logs are being rotated that way currently and it would be a mess to try and change that